src/page-response-impact/ValidDataWrapper.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { sumAgeGroupPopulations } from "../impact-dashboard/EpidemicModelContext";
+import { Facilities } from "../page-multi-facility/types";
+const POPULATION_DATA_ERROR_MSG =
+  "Impact could not be generated because one or more of your facilities does not have an incarcerated population count. Please add population numbers and try again.";
+const LOCALE_DATA_ERROR_MSG =
+  "Impact could not be generated because locale information does not match across facilities.";
+
+const ValidDataWrapperContainer = styled.div``;
+const ErrorMessage = styled.div`
+  padding: 20px;
+  color: ${Colors.darkForest};
+  width: 60%;
+`;
+
+interface Props {
+  children?: React.ReactElement<any>;
+  facilities?: Facilities;
+}
+
+const ValidDataWrapper: React.FC<Props> = ({ children, facilities = [] }) => {
+  const [errors, setErrors] = useState({
+    locale: false,
+    population: false,
+  });
+
+  function validateLocaleData(facilities: Facilities) {
+    return facilities.some((f) => {
+      return (
+        facilities[0].modelInputs.stateCode !== f.modelInputs.stateCode ||
+        facilities[0].systemType !== f.systemType
+      );
+    });
+  }
+
+  function validatePopulationData(facilities: Facilities) {
+    return facilities.some((f) => {
+      return sumAgeGroupPopulations(f) === 0;
+    });
+  }
```

Comment:
I chose to iterate over the facilities twice here just for clarify and ability to easily understand what the valid cases are since we are moving quickly and don't have tests. I don't really have a concept of how many facilities these systems have, so if this is causing a performance issue it will be easy to refactor.